templates/virtual_uc_topology.yml.j2

```diff
@@ -0,0 +1,25 @@
+name: undercloud
+prefix: null
+cpu: "8"
+cpumodel: "host-passthrough"
+memory: "24576"
+swap: "0"
+deploy_os: true
+os:
+    type: "linux"
+    variant: "rhel7"
```

Comment:
This needs to change, please parameterize this based on OSP version. rhel8 for OSP 15 and above.